<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="com.erp.automation.testUtilsListeners.TestListener"/>
  </listeners>
  <test thread-count="5" name="Test1111(failed)">
    <classes>
      <class name="com.erp.automation.tests.purchase.PoCloseProcessTest">
        <methods>
          <include name="afterMethod"/>
          <include name="afterClass"/>
          <include name="loginToApplication"/>
          <include name="poCloseProcessTest"/>
          <include name="launchErpSite"/>
        </methods>
      </class> <!-- com.erp.automation.tests.purchase.PoCloseProcessTest -->
      <class name="com.erp.automation.tests.purchase.PoRejectProcessTest">
        <methods>
          <include name="launchErpSite"/>
          <include name="PoApprovalProcessTest"/>
          <include name="afterMethod"/>
          <include name="afterClass"/>
          <include name="loginToApplication"/>
        </methods>
      </class> <!-- com.erp.automation.tests.purchase.PoRejectProcessTest -->
    </classes>
  </test> <!-- Test1111(failed) -->
</suite> <!-- Failed suite [Suite] -->
